Sources/MapboxMaps/Foundation/Extensions/Core/OfflineRegionManager.swift

```diff
@@ -38,4 +38,14 @@ extension OfflineRegionManager {
                                                                                     type: OfflineRegion.self,
                                                                                     concreteErrorType: MapError.self))
     }
+
+    // :nodoc:
+    public func invalidateOfflineRegion(_ offlineRegion: OfflineRegion, completion: @escaping (Error?) -> Void) {
+        offlineRegion.invalidate(forCallback: coreAPIClosureAdapter(for: completion, concreteErrorType: MapError.self))
```

Comment:
Could these methods be made public as a part of `OfflineRegion` instead? It could be more handy to be able to call this these methods directly on a region without having to drag a manager around for this.

Reply:
It's an interesting proposal.
My point was to extend swift classes we own in the MapboxMaps (and `[MBM]OfflineRegion` is bindgen-generated one).
WDYT @macdrevx ?

Reply:
I support this. No reason to add API to OfflineRegionManager. Let's implement these as refinements on OfflineRegion.